modules/appeals_api/spec/requests/v1/notice_of_disagreements_controller_spec.rb

```diff
@@ -216,7 +216,7 @@ def base_path(path)
       uuid = create(:notice_of_disagreement).id
       get("#{path}#{uuid}")
       expect(response.status).to eq(200)
-      expect(parsed.dig('data', 'attributes', 'formData')).to be_a Hash
+      expect(parsed['data']['attributes'].key?('form_data')).to be false
```

Comment:
The serializers are not returning form_data, not even a form_data key with a nil value, so check to make sure not form_data key is present

Reply:
@cilestin should i keep this expect, going after the form_data key now

Reply:
Yeah it's fine to keep in, IMO. Better to catch ourselves and be intentional.